src/main/java/core/basesyntax/service/impl/StorageUpdaterServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.StorageUpdaterService;
+import core.basesyntax.strategy.StorageService;
+import core.basesyntax.strategy.StorageStrategy;
+import java.util.List;
+
+// IDK what is better: either have this class or transfer this logic to StorageStrategy????
+public class StorageUpdaterServiceImpl implements StorageUpdaterService {
+    private StorageStrategy strategy = new StorageStrategy();
```

Comment:
Not sure myself, but I think it's ok as it is
also to implement dependancy injection lets initialize class filed in constructor